library/src/main/scala/g8.scala

```diff
@@ -82,14 +100,21 @@ object G8 {
   }
 
   def write(in: File, out: File, parameters: Map[String, String], append: Boolean) {
-    Option(PlexusIoResourceAttributeUtils.getFileAttributes(in)) match {
-      case Some(attr) =>
-        val mode = attr.getOctalMode
-        write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
-        util.Try(ArchiveEntryUtils.chmod(out, mode, new ConsoleLogger(Logger.LEVEL_ERROR, "")))
-      case None =>
-        // PlexusIoResourceAttributes is not available for some OS'es such as windows
-        write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+    try {
+      Option(PlexusIoResourceAttributeUtils.getFileAttributes(in)) match {
+        case Some(attr) =>
+          val mode = attr.getOctalMode
+          write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+          util.Try(ArchiveEntryUtils.chmod(out, mode, new ConsoleLogger(Logger.LEVEL_ERROR, "")))
+        case None =>
+            // PlexusIoResourceAttributes is not available for some OS'es such as windows
+            write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+      }
+    }
+    catch {
+      case t : Throwable =>
+        println("Falling back to file copy for %s: %s" format(in.toString, t.getMessage))
```

Comment:
1. Only if we're not reducing the amount of useful error messages if any. 
2. Not sure what others (like @xuwei-k) might say, but I personally don't think it's a good idea to fall back to file copying in case there's an actual error.